python/lsst/meas/astrom/astrometry.py

```diff
@@ -55,6 +57,11 @@ class AstrometryConfig(RefMatchConfig):
         default=0.001,
         min=0,
     )
+    doWriteOutput = pexConfig.Field(
```

Comment:
I feel like a better name for this config parameter would be `doOutputUsedFlag`.